packages/orchestrator-ui-components/package.json

```diff
@@ -24,7 +24,7 @@
     "scripts": {
         "test": "jest",
         "generate-version": "node -p \"'export const ORCHESTRATOR_UI_LIBRARY_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/configuration/version.ts && prettier --write src/configuration/version.ts",
-        "build": "npm run generate-version && tsup src/index.ts --format esm --dts --tsconfig tsconfig.build.json",
+        "build": "npm run generate-version && tsup src/index.ts",
```

Comment:
What does this do? Or why was it there?

Reply:
The removed parameters are moved to a separate file now `tsup.config.ts`